quickwit-core/src/indexing/index.rs

```diff
@@ -79,8 +80,8 @@ pub async fn index_data(
     }
 
     let document_retriever = Box::new(DocumentSource::create(&params.input_uri).await?);
-    let (statistic_collector, statistic_sender) = StatisticsCollector::start_collection();
     let (split_sender, split_receiver) = channel::<Split>(SPLIT_CHANNEL_SIZE);
+    let (task_completed_sender, task_completed_receiver) = watch::channel::<bool>(false);
```

Comment:
can we move that in the call in the index_data_cli function?

Reply:
The rationale is that this function could then be used by a different client that the quickwit-cli